src/harness/testcases/testdata/esc_zone_contains_device_a.json

```diff
@@ -0,0 +1,18 @@
+{
+ "id" : "zone/exclusion_zone/ntia/2017_04_10/esc_zone_contains_deviceA_harness_test",
+ "creator" : "SAS_Administrator_Test",
+ "usage" : "EXCLUSION_ZONE",
+ "zone" :
+ {
+  "type": "Polygon",
+  "coordinates":
+  [
+   [
+    [37.525056, -122.284112],
```

Comment:
FYI device A has moved to Kansas to make other test cases easier. It's probably best to define an ESC zone along the coast and then move device A inside or nearby as needed.